b2luigi/core/utils.py

```diff
@@ -229,6 +230,12 @@ def create_output_file_name(task, base_filename, result_dir=None):
         # Be sure to evaluate things relative to the current executed file, not to where we are now
         result_dir = map_folder(get_setting("result_dir", task=task, default=".", deprecated_keys=["result_path"]))
 
+    for key, value in serialized_parameters.items():
```

Comment:
Now that I included a for loop over the parameters, I could also build the `param_list` in the same loop instead of the separate list comprehension below.